blockchain/genesis/genesis.go

```diff
@@ -321,6 +323,7 @@ type (
 		WithdrawWaitingPeriod time.Duration        `yaml:"withdrawWaitingPeriod"`
 		MinStakeAmount        string               `yaml:"minStakeAmount"`
 		BootstrapCandidates   []BootstrapCandidate `yaml:"bootstrapCandidates"`
+		ContractAddress       string               `yaml:"contractAddress"`
```

Comment:
move to `genesis.Poll`, there is a `NativeStakingContractAddress` there, you can use `LSDStakingContractAddress`